app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  def post_update_counter
+    user = User.find(author_id)
+    user.update_attribute(:posts_count, user.posts_count + 1)
+  end
+
+  def most_recent_comments
+    Comment.where(post_id: id).order(created_at: :desc).limit(3)
```

Comment:
The Post model should have a method that returns the _**5**_ most recent comments for a given post. You are returning _**3,**_ please correct this.